python_modules/dagster/dagster/_core/definitions/run_request.py

```diff
@@ -352,6 +356,48 @@ def __new__(
         )
 
 
+@whitelist_for_serdes
+@experimental
+class BackfillRequest(
+    NamedTuple(
+        "_BackfillRequest",
+        [
+            ("asset_partitions", "AssetGraphSubset"),
+            ("tags", PublicAttr[Optional[Mapping[str, str]]]),
+        ],
+    )
+):
```

Comment:
Would be helpful to add a docstring that emphasizes this class isn't meant to be directly instantiated.

Would prefer to keep `AssetGraphSubsets` and `PartitionsSubsets` internal concepts since these are still fluctuating.

Reply:
Good point - adding.